prac_05/state_names1.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+"""LBYL"""
+
+state = input("Enter short state: ").upper()
+while state != "":
+    if state in CODE_TO_NAME:
+        print(state, "is", CODE_TO_NAME[state])
+    else:
+        print("Invalid short state")
+    state_code = input("Enter short state: ").upper()
```

Comment:
Ensure consistent variable naming. In the first loop, you use "state", and in the second loop, you use "state_code"